src/main/java/uk/gov/hmcts/sptribs/ciccase/model/OrderTemplate.java

```diff
@@ -22,7 +22,10 @@ public enum OrderTemplate implements HasLabel {
     CIC10_STRIKE_OUT_WARNING("CIC10 - Strike Out Warning", "CIC10_Strike_Out_Warning"),
 
     @JsonProperty("CIC13_Pro_Forma_Summons")
-    CIC13_PRO_FORMA_SUMMONS("CIC13 - Pro Forma Summons", "CIC13_Pro_Forma_Summons");
+    CIC13_PRO_FORMA_SUMMONS("CIC13 - Pro Forma Summons", "CIC13_Pro_Forma_Summons"),
+
+    @JsonProperty("CIC14 – LO General Directions")
```

Comment:
Do we want the Json property to follow the snake_case naming policy used elsewhere in this class?

Reply:
yes, good catch